modules/components/src/AdvancedFacetView/Facets/TermAggregation.js

```diff
@@ -2,24 +2,31 @@ import React from 'react';
 import TermAggs from '../../Aggs/TermAgg';
 import { inCurrentSQON, replaceSQON, toggleSQON } from '../../SQONView/utils';
 
-export default ({ aggType, aggProps, title, onValueChange, sqon, path }) => (
-  <TermAggs
-    key={title}
-    collapsible={false}
-    handleValueClick={({ bucket }) => {
-      onValueChange({
-        value: bucket.key,
-      });
-    }}
-    field={path}
-    buckets={aggProps ? aggProps.buckets || [] : []}
-    displayName={title}
-    isActive={d => {
-      return inCurrentSQON({
-        value: d.value,
-        dotField: d.field,
-        currentSQON: sqon,
-      });
-    }}
-  />
-);
+export default ({ aggType, aggProps, title, onValueChange, sqon, path }) => {
+  const buckets = aggProps ? aggProps.buckets || [] : [];
```

Comment:
can this be 

```
aggProps?.buckets || []
````

Reply:
oh yes... don't why I didn't think of that :P